content/blog/2021/04/2021-04-12-jenkins-operator-sub-project.adoc

```diff
@@ -0,0 +1,57 @@
+
+== What does it mean for the project?
+
+image:/images/logos/kubernetes-operator/256.png["Jenkins Operator", role=left]
+
+Becoming an official part of the Jenkins project was a major step towards better alignment with the overall Jenkins’ roadmap and more opportunities to increase adoption of the Jenkins Operator project.
+
+Finally, with a dedicated team at link:http://virtuslab.com/[VirtusLab] actively maintaining to the project we can engage with the wider community and participate in some of the Cloud-Native SIG meetings.
+*This opens a room for everyone to voice their opinions or start supporting the project.*
+
+We truly believe that this community engagement will result in significant improvements to Jenkins Operator, as well as Jenkins ecosystem itself.
+
+== Bridging the gap between Jenkins and Kubernetes
+
+Running Jenkins in a cloud-native environment like Kubernetes is not a trivial task.
+With Jenkins Operator project we want to enable the community to take full advantage of Kubernetes and public cloud capabilities by:
+
+* native integration with public cloud services in areas of observability, storage and cloud security
+* Kubernetes autoscaling and self-healing mechanism
+* secure access to Jenkins instance
+* declarative configuration using Kubernetes Custom Resources
+* full lifecycle management, eventually transforming it to autopilot
+
+== Take part in the journey toward global trends
+
+*Start link:https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md[contributing] and play an important role in creating the automated Jenkins experience! Don’t hesitate - engage in the community.*
+Join us in our work of creating the functionalities you see beneficial.
+You are welcome to create issues and pull requests. We are actively resolving community issues and providing answers on a dedicated Slack channel.
+
+As the project has been initially developed and still actively maintained by link:http://virtuslab.com/[VirtusLab], we started link:https://groups.google.com/g/jenkinsci-dev/c/OA5nb_SAgh0/m/OoBS2o8nAwAJ:[discussion] towards open governance model to facilitate communication and collaboration.
+
+== Lead the way we are heading by providing us feedback
+
+To celebrate the new possibilities that are being opened for our project, we would like to invite you to take part in a short survey that will help put us on the right track.
+If you have been using Jenkins Operator or running Jenkins in any other environment, please take a moment to fill out our quick survey.
+*We will choose at least three of the most informative answers and send you an awesome Jenkins Operator T-shirt with our cute Gopher butler.*
+The survey can be found link:https://docs.google.com/forms/d/1doIkgnm3_WbjtlwWSU4sOoiI7QoneHlYIjXEJOVMrfQ/edit?usp=sharing[here]. Remember, the most sincere answers are the best.
```

Comment:
I'm a little concerned that the survey includes mandatory fields for name and email address.  I think that will make it difficult to share the full results of the survey and might be considered more personal information than survey respondents are ready to provide.

Could the name and email fields be made optional?

![screencapture-docs-google-forms-d-1doIkgnm3-WbjtlwWSU4sOoiI7QoneHlYIjXEJOVMrfQ-viewform-2021-04-14-07_06_14-edit](https://user-images.githubusercontent.com/156685/114715633-8311d480-9cf0-11eb-809e-8655bee70110.png)

Reply:
absolutely! valid points, thank you 

I'll make them optional with a note that responses without an email address are not eligible for a T-shirt